pom.xml

```diff
@@ -42,7 +42,7 @@
         <module>plugins</module>
         <module>serving-api</module>
         <module>doc</module>
-        <module>web</module>
+	<!-- <module>web</module> -->
```

Comment:
The project do not compile in my local if these lines are not commented. Was not it fixed?